projects/auth0-angular/src/lib/auth.module.ts

```diff
@@ -8,15 +8,20 @@ import { AuthGuard } from './auth.guard';
 @NgModule()
 export class AuthModule {
   static forRoot(config: AuthConfig): ModuleWithProviders<AuthModule> {
+    const defaultedConfig: AuthConfig = {
```

Comment:
nit: can we rename this to `defaultConfig`, `defaultValues` or even `baseConfig` or `baseValues` ?

Reply:
Sure, I'm not precious about the name.